src/Resources/contao/modules/ModuleArticle.php

```diff
@@ -366,7 +366,7 @@ public function generatePdf()
 		// Close and output PDF document
 		$pdf->lastPage();
 		$pdf->Output(standardize(ampersand($this->title, false)) . '.pdf', 'D');
-
+		// FIXME: This has to be changed to a ResponseException somehow - rather write to temp file and stream it then?
```

Comment:
The `BinaryFileResponse` has to be used here as in 99.9% of all our cases. I don't have a valid use case for `StreamedResponse` in Contao, yet.